src/app/_components/Icon.tsx

```diff
@@ -1,4 +1,4 @@
-import { type Icon } from '@phosphor-icons/react'
+import type { Icon } from '@phosphor-icons/react'
```

Comment:
when do we put `type` outside of the bracket, and when inside? They are both allowed, just curious if we should follow any pattern?

Reply:
We cam use `import type { TypeName }` when importing only types to clearly indicate no runtime code is being imported. For mixed imports, we can use `import { type TypeName, AnotherItem }` to keep related imports together and maintain organized code.